common/src/main/java/org/mvndaemon/mvnd/common/logging/TerminalOutput.java

```diff
@@ -279,8 +281,8 @@ private boolean doAccept(Message entry) {
             Project prj = projects.remove(artifactId);
             if (prj != null) {
                 prj.log.forEach(log);
+                doneProjects.add(artifactId);
```

Comment:
Why do we need `doneProjects` set? Under which circumstances would deliver a different count than the original `int doneProjects`?